pyomo/repn/plugins/nl_writer.py

```diff
@@ -2268,8 +2372,45 @@ class text_nl_debug_template(object):
     _create_strict_inequality_map(vars())
 
 
+nl_operators = {
```

Comment:
Is this in a "sane to Siirola" order?

Reply:
It matches the forward template order above.

Reply:
Gotcha gotcha.